youtube-client/src/index.scss

```diff
@@ -0,0 +1,55 @@
+* {
```

Comment:
I don't see any benefits in combining scss and styled-components, unless you'd like to try it together intentionally. It seems as a better option to reduce number of tools in real project